Region/Histogram.cc

```diff
@@ -13,40 +13,47 @@ Histogram::Histogram(int num_bins, float min_value, float max_value, const std::
 
 Histogram::Histogram(Histogram& h, tbb::split) : _bin_width(h._bin_width), _min_val(h._min_val), _hist(h._hist.size(), 0), _data(h._data) {}
 
-void Histogram::operator()(const tbb::blocked_range<size_t>& r) {
-    std::vector<int> tmp(_hist);
+void Histogram::operator()(const tbb::blocked_range<uint64_t>& r) {
+    std::vector<uint64_t> tmp(_hist);
```

Comment:
Do we need to store the bins as 64bit? They get sent as 32bit ints anyway, don't they?

Reply:
Fixed